src/site/_data/courses/example/meta.yml

```diff
@@ -0,0 +1,21 @@
+# The date is used to organize the courses on the /learn page.
+# This should be the initial publish date of the course. When new pages are
+# added to the course the site will check if they are <30 days old, and if they
+# are, it will give them a "New" badge. However, when you first launch a course
+# you don't want every entry to have a "New" badge, so anything that existed
+# on or before this date will not receive the "New" badge.
+date: 2022-01-17
+url: /handbook/content-types/example-course/
+draft: false
+# When true the course will not be listed on the /learn page
+invisible: true
+title: i18n.courses.example.title
+description: i18n.courses.example.description
+# This is the logo that appears in the navigation drawer and header.
+logo: '/images/courses/accessibility/logo.svg'
+# This is used to provide the background for the navigation drawer and header.
+background: '/images/courses/accessibility/background.svg'
+# This provides the cover image for the card on /learn/
+card: '/images/courses/accessibility/card.svg'
```

Comment:
I wonder if we should use specific example assets here instead? Now whenever (if ever) the a11y assets change, our test page would also change.